test/test_compiler.js

```diff
@@ -10409,6 +10409,42 @@ p_recip();`,
   } finally {
     await __env.exitProcedure(2, null);
   }`]],
+
+
+    // 110
+    [`attimer(time=[new Time(12, 30)], once=true) => @org.thingpedia.builtin.thingengine.builtin.say(message="hey");`,
+    [`"use strict";
+  let _t_0;
+  let _t_1;
+  let _t_2;
+  let _t_3;
+  let _t_4;
+  let _t_5;
+  let _t_6;
+  try {
+    _t_1 = new Array(1);
+    _t_2 = new __builtin.Time(12, 30, 0);
+    _t_1[0] = _t_2;
+    _t_0 = await __env.invokeAtTimer(_t_1, null);
```

Comment:
Yeah see: there is nothing in this call to invokeAtTimer that passes the once=true, because "once=true" got lost in the translation from Ast.Expression to Ast.Table